pkg/features/kube_features.go

```diff
@@ -990,7 +990,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	JobPodReplacementPolicy: {Default: false, PreRelease: featuregate.Alpha},
 
-	JobReadyPods: {Default: true, PreRelease: featuregate.Beta},
+	JobReadyPods: {Default: true, PreRelease: featuregate.Beta, LockToDefault: true},
```

Comment:
```suggestion
	JobReadyPods: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

I think this is being GA so we should update the PreRelease also.


/ok-to-test
/sig apps

Reply:
fixed